firebase/README.md

```diff
@@ -2,15 +2,15 @@
 
 ## Setup
 
-You will need to set up a new firebase project and enable the following features:
+You will need to set up a new firebase project (https://console.firebase.google.com) and enable the following features:
```

Comment:
Make this a link?

Reply:
I prefer to see the link address in situations like this, so you're sure where it will take you.
Can change if you disagree.